package.yaml

```diff
@@ -42,19 +47,19 @@ dependencies:
   - wai
   - wai-middleware-metrics
   - wai-cli
-  - servant
+  # - servant
```

Comment:
woah wait what?! servant is like the main thing here

Reply:
It surprised me too, but `servant-server` is where the `Servant` module is actually defined, and it re-exports everything Magicbane directly uses. Granted it depends on `servant` anyway so this isn't saving any build time.

Reply:
oh, right — I didn't even notice the `servant-server` on the next line. facepalm.